### PR TITLE
create new post clean up old

### PR DESCRIPTION
**Pull request by ksatx**
Creating new blog post and removing convertful HTML tag from a few older ones.

### CODE REVIEW

**Comment by lottamus** [comment]
Deploy preview for *stoplightio* ready!

Built with commit 1526703a2021c4202e8ac015c0caa0365e845d3b

https://deploy-preview-1353--stoplightio.netlify.com

File: netlify/blog-posts/companies-supporting-open-source.md [hunk 1]
```diff
@@ -0,0 +1,129 @@
+---
+path: /blog/companies-supporting-open-Source
```
Comment (philsturgeon) [inline netlify/blog-posts/companies-supporting-open-source.md]:
```suggestion
path: /blog/companies-supporting-open-source
```

**Comment by ksatx** [comment]
[Phil Blog Post](https://trello.com/c/shQDjlJy/373-phil-blog-post)

File: netlify/blog-posts/companies-supporting-open-source.md [hunk 2]
```diff
@@ -0,0 +1,129 @@
...
+title: >-
+  Companies Supporting Open-Source
+subtitle: >-
+  A look at the ways companies use and support open-source projects.
+image: /images/supporting-open-source.jpg
+includeToc: false
+actionBar:
+  enabled: false
+meta:
+  url: https://stoplight.io/blog/companies-supporting-open-Source/
```
Comment (philsturgeon) [inline netlify/blog-posts/companies-supporting-open-source.md]:
```suggestion
  url: https://stoplight.io/blog/companies-supporting-open-source/
```

File: netlify/blog-posts/companies-supporting-open-source.md [hunk 3]
```diff
@@ -0,0 +1,129 @@
...
+    Companies Supporting Open-Source
+  image: /images/supporting-open-source.jpg
+  twitter:
+    title: >-
+      Stoplight Blog: Companies Supporting Open-Source
+    image: /images/supporting-open-source.jpg
+    username: '@stoplightio'
+---
+
+Companies like Stoplight stand on the shoulders of giants, by combining existing open-source tooling, built around freely available open standards, to produce a combination of open and closed source code. We don't want to just take from open-source, we want to support the projects we use, and there are two main ways that companies can do that.
```
Comment (philsturgeon) [inline netlify/blog-posts/companies-supporting-open-source.md]:
```suggestion
Companies like Stoplight stand on the shoulders of giants, by combining existing open-source tooling, built around freely available open standards, to produce a combination of open and closed source code. We don't want to just take from open-source, we want to support the projects we use, and there are three main ways that companies can do that.
```

Comment (philsturgeon) [reply netlify/blog-posts/companies-supporting-open-source.md]: Hahaha going to sleep now I’m sure the rest is right 🤦‍♂️

**Comment by philsturgeon** [comment]
@paulatulis I'll leave this one with you! Can you click merge if you feel like it's about right? No worries if some of my typos sneak through, they're often well hidden.

**Comment by collinbachi** [comment]
I had to roll this back--for whatever reason it was throwing 404's on the marketing site

**Pull request merged**